.docker/Dockerfile

```diff
@@ -227,3 +227,68 @@ RUN echo "if [ -f ${WORKSPACE}/install/setup.bash ]; then source ${WORKSPACE}/in
     # Expose the environment variables to the non-root user
     && echo "if [ -f /sim.sh ]; then source /sim.sh; fi" >> /home/$USERNAME/.bashrc \
     && echo "if [ -f /dev.sh ]; then source /dev.sh; fi" >> /home/$USERNAME/.bashrc
+
+
+######################################################################################
```

Comment:
This is pretty redundant with the above stage. Though I would like to ultimately see this templated, so I'll leave it for now and open a new issue to think of a better solution

Reply:
See #60